### PR TITLE
Reformating Issue #946

### PR DESCRIPTION
**Pull request by j0rt3g4**
I've done the reformat to github markdown as well as added a note for the Onedrive variables.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/usmt/usmt-recognized-environment-variables.md [hunk 1]
```diff
@@ -18,443 +18,126 @@ When using the XML files MigDocs.xml, MigApp.xml, and MigUser.xml, you can use e
...
 
--   [Variables that are processed for the operating system and in the context of each user](#bkmk-1)
+-   [Variables that are processed for the operating system and in the context of each user](#Variables-that-are-processed-for-the-operating-system-and-in-the-context-of-each-user)
 
--   [Variables that are recognized only in the user context](#bkmk-2)
+-   [Variables that are recognized only in the user context](#Variables-that-are-recognized-only-in-the-user-context)
 
-## <a href="" id="bkmk-1"></a>Variables that are processed for the operating system and in the context of each user
+>[!IMPORTANT]
+>Onedrive and Sharepoint variables are considered different products that's the reason why are not listed here, on the windows 10 variables.
```
Comment (JohanFreelancer9) [inline windows/deployment/usmt/usmt-recognized-environment-variables.md]:
```suggestion
>Onedrive and Sharepoint variables are considered different products, that is why they are not listed here as Windows 10 variables.
```

File: windows/deployment/usmt/usmt-recognized-environment-variables.md [hunk 2]
```diff
@@ -18,443 +18,126 @@ When using the XML files MigDocs.xml, MigApp.xml, and MigUser.xml, you can use e
...
-<td align="left"><p><strong>WINDIR</strong></p></td>
-<td align="left"><p>Refers to the Windows folder located on the system drive.</p></td>
-</tr>
-</tbody>
-</table>
-
- 
-
-## <a href="" id="bkmk-2"></a>Variables that are recognized only in the user context
+| Variable  | Explanationr |
```
Comment (JohanFreelancer9) [inline windows/deployment/usmt/usmt-recognized-environment-variables.md]:
```suggestion
| Variable  | Explanation |
```

File: windows/deployment/usmt/usmt-recognized-environment-variables.md [hunk 3]
```diff
@@ -18,443 +18,126 @@ When using the XML files MigDocs.xml, MigApp.xml, and MigUser.xml, you can use e
...
+| :---: | ------------- |
+|ALLUSERSAPPDATA |Same as CSIDL_COMMON_APPDATA|
```
Comment (JohanFreelancer9) [inline windows/deployment/usmt/usmt-recognized-environment-variables.md]:
```suggestion
|ALLUSERSAPPDATA |Same as CSIDL_COMMON_APPDATA.|
```

File: windows/deployment/usmt/usmt-recognized-environment-variables.md [hunk 4]
```diff
@@ -18,443 +18,126 @@ When using the XML files MigDocs.xml, MigApp.xml, and MigUser.xml, you can use e
...
+|ALLUSERSPROFILE |Refers to %PROFILESFOLDER%\Public or %PROFILESFOLDER%\all users.|
+|COMMONPROGRAMFILES |Same as CSIDL_PROGRAM_FILES_COMMON.|
+|COMMONPROGRAMFILES(X86)| Refers to the C:\Program Files (x86)\Common Files folder on 64-bit systems.|
+|CSIDL_COMMON_ADMINTOOLS|Version 10.0. The file-system directory that contains administrative tools for all users of the computer.|
+|CSIDL_COMMON_ALTSTARTUP|The file-system directory that corresponds to the non-localized Startup program group for all users.|
+|CSIDL_COMMON_APPDATA|The file-system directory that contains application data for all users. A typical path Windows is C:\ProgramData.|
```
Comment (JohanFreelancer9) [inline windows/deployment/usmt/usmt-recognized-environment-variables.md]:
```suggestion
|CSIDL_COMMON_APPDATA|The file-system directory that contains application data for all users. A typical Windows path is C:\ProgramData.|
```

File: windows/deployment/usmt/usmt-recognized-environment-variables.md [hunk 5]
```diff
@@ -18,443 +18,126 @@ When using the XML files MigDocs.xml, MigApp.xml, and MigUser.xml, you can use e
...
-<p>S-1-5-21-1714567821-1326601894-715345443-1026.</p></td>
-</tr>
-</tbody>
-</table>
-
- 
+| Variable  | Explanation |
+| :---: | ------------- |
+|APPDATA|Same as CSIDL_APPDATA.|
+|CSIDL_ADMINTOOLS|The file-system directory that is used to store administrative tools for an individual user. The Microsoft® Management Console (MMC) saves customized consoles to this directory, which roams with the user profile.|
```
Comment (JohanFreelancer9) [inline windows/deployment/usmt/usmt-recognized-environment-variables.md]:
```suggestion
|CSIDL_ADMINTOOLS|The file-system directory that is used to store administrative tools for an individual user. The Microsoft Management Console (MMC) saves customized consoles to this directory, which roams with the user profile.|
```

File: windows/deployment/usmt/usmt-recognized-environment-variables.md [hunk 6]
```diff
@@ -18,443 +18,126 @@ When using the XML files MigDocs.xml, MigApp.xml, and MigUser.xml, you can use e
...
+|CSIDL_DRIVES|The virtual folder representing My Computer that contains everything on the local computer: storage devices, printers, and Control Panel. The folder may also contain mapped network drives.|
+|CSIDL_FAVORITES|The file-system directory that serves as a common repository for the user's favorites. A typical path is C:\Users\Username\Favorites.|
+|CSIDL_HISTORY|The file-system directory that serves as a common repository for Internet history items.|
+|CSIDL_INTERNET|A virtual folder for Internet Explorer.|
+|CSIDL_INTERNET_CACHE|The file-system directory that serves as a common repository for temporary Internet files. A typical path is C:\Users\username\AppData\Local\Microsoft\Windows\Temporary Internet Files|
+|CSIDL_LOCAL_APPDATA|The file-system directory that serves as a data repository for local, non-roaming applications. A typical path is C:\Users\username\AppData\Local.|
+|CSIDL_MYDOCUMENTS|The virtual folder representing My Documents.A typical path is C:\Users\Username\Documents.|
```
Comment (JohanFreelancer9) [inline windows/deployment/usmt/usmt-recognized-environment-variables.md]:
```suggestion
|CSIDL_MYDOCUMENTS|The virtual folder representing My Documents. A typical path is C:\Users\Username\Documents.|
```

**Pull request closed without merging**